### PR TITLE
[misspelling] Fix some misspells - names/descriptions

### PR DESCRIPTION
**Pull request by Kri5t0fK**
Those are things I've found some time ago - better to PR them already before I'd need to recheck ;)
- a couple of misspells in names/descriptions,
- also some description improvements.

### CODE REVIEW

File: data/bingo/functions/init/init.mcfunction [hunk 1]
```diff
@@ -48,13 +48,13 @@ forceload add 0 0
...
+	# weight: (Integer) Weight of this item for the RNG.
 	#
 	# Categories have the following structure:
-	# name: (String) Namespaced id to uniquly identify the category within bingo
-	# translateableName: (String) Text component used for displaying the category's
+	# name: (String) Namespaced id to uniquely identify the category within bingo
+	# translatableName: (String) Text component used for displaying the category's
 	# 	name
-	# items: (Compund List) (readonly) List of all items this category has. Entries
+	# items: (Compound List) (read-only) List of all items this category has. Entries
```
Comment (NeunEinser) [inline data/bingo/functions/init/init.mcfunction:57]: `readonly` is a typical keyword in programming and should be spelled as one word

File: data/bingo/functions/init/init.mcfunction [hunk 2]
```diff
@@ -48,13 +48,13 @@ forceload add 0 0
 	# 	inventory. Should look like this: "clear @a[tag=bingo.clear] <item> 1
 	# categories: (String List) List of category names. All categories referenced
 	# 	here should be registered separately.
-	# weight: (Integer) Weight of this item for the rng.
+	# weight: (Integer) Weight of this item for the RNG.
```
Comment (NeunEinser) [inline data/bingo/functions/init/init.mcfunction:51]: This one is especially philosophical and in doc comment doesn't matter. No need to undo it, just also no need to think about what the "more correct" capitalization might be.

File: data/bingo/functions/init/init.mcfunction [hunk 3]
```diff
@@ -943,9 +945,9 @@ summon minecraft:area_effect_cloud 0 0 0 {CustomName:'{"translate": "bingo.techn
 		#declare score_holder $is_multiplayer
 		scoreboard players add $is_multiplayer bingo.state 0
 		#>
-		# The status of pregeneration.
+		# The status of chunk pregeneration.
 		# 0 = not started
-		# 1 = started
+		# 1 = started / during the process
 		# 2 = completed
```
Comment (NeunEinser) [inline data/bingo/functions/init/init.mcfunction:951]: Isn't this redundant / the same?

**Review by NeunEinser — COMMENTED** [review]
In general, I would like to point out that I don't want people specifically looking for things like this.

If you made this an issue with a list of things, I would close it as Invalid/wont fix, as I myself really don't care and it doesn't actually improve / change anything for the end user.

Of course, if you see a typo by just looking at code and want to fix it, you can do so, but in case you are specifically reviewing doc comments and declarations, this would not be a good use of your time, also and perhaps even especially, if this means you would do something not related to Bingo otherwise.

**Comment by NeunEinser** [comment]
Oh and yeah, in case of translation files, this is of course different, as what's contained there actually does change sth for the end user 😉

File: data/bingo/functions/init/init.mcfunction (hunk 3 above)
Comment (NeunEinser) [reply data/bingo/functions/init/init.mcfunction:951]: (see what I did there? 😜)

**Review by Kri5t0fK — COMMENTED** [review]
I basically found those misspells while searching through the doc (and understanding implementation of some things) a couple of months ago and just put them on separate branch, then forgot about it.

So yeah - in the future, I won't do it as a separate branch, instead incorporate those as small parts of code PRs ;)

**Review by NeunEinser — APPROVED** [review]

**Pull request merged**